src/main/java/com/hermesworld/ais/galapagos/events/GalapagosEventSink.java

```diff
@@ -43,4 +44,8 @@
 
     CompletableFuture<Void> handleApplicationOwnerRequestCanceled(ApplicationOwnerRequest request);
 
+    CompletableFuture<Void> handleAddTopicProducers(List<String> producers, TopicMetadata metadata);
+
+    CompletableFuture<Void> handleRemoveProducerFromTopic(TopicMetadata metadata, String appDeleteId);
```

Comment:
Inconsistent naming. Either remove "FromTopic" here, or add "ToTopic" to the method above. Also, "appDeleteId" is a bad parameter name for a public interface. And (third), parameter order is inconsistent between the two methods (TopicMetadata first or second?)